libobs/cmake/os-freebsd.cmake

```diff
@@ -1,17 +1,18 @@
 find_package(X11 REQUIRED)
 find_package(x11-xcb REQUIRED)
 # cmake-format: off
-find_package(xcb OPTIONAL_COMPONENTS xcb-xinput QUIET)
+find_package(xcb REQUIRED xcb OPTIONAL_COMPONENTS xcb-xinput)
```

Comment:
The non-legacy CMake codepath relies on CMake's own dependency resolution to report missing dependencies. The legacy code choses to suppress CMake's own errors and instead display custom error messages, unnecessarily adding code complexity.

In fact the legacy code would potentially configure the project without issue at first, but then throw errors at build generation because `XCB::XCB` was added as a dependency but the target definition could not be resolved.

Reply:
Got it.